README.md

```diff
@@ -2,3 +2,39 @@ BigQuery ETL
 ===
 
 Bigquery UDFs and SQL queries for building derived datasets.
+
+Recommended practices
+===
+
+- Should name sql files like `sql/destination_table_with_version.sql` e.g.
+  `sql/clients_daily_v6.sql`
+- Should not specify a project or dataset in table names to simplify testing
+- Should use incremental queries
+- Should filter input tables on partition and clustering columns
+- Should use UDF language `SQL` over `js` for performance
+- Should use UDFs for reusability
```

Comment:
> Is it understood yet how to reuse SQL UDFs

right now we have to prepend them to the query, e.g. `cat udfs/get_key.sql sql/my_query.sql | bq query`.

> Airflow has params for udf configuration, but that appears to be relevant only to legacy SQL.

The docs made it look like this might have changed, but testing confirms that it doesn't work with standard SQL. Maybe some day.